packages/kbn-monaco/src/yaml/index.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Co-locate yaml implementation for code editor within kbn-monaco, similar to how other languages are setup but have it load on demand.